modules/ROOT/pages/servicenow/servicenow-connector.adoc

```diff
@@ -2,59 +2,48 @@
+See the xref:release-notes::connector/servicenow-connector-release-notes-mule-4.adoc[ServiceNow Connector Release Notes] for information about compatibility. 
+
 
 == About Connectors
 
-Anypoint connectors are Mule runtime engine extensions that enable you to connect
-to APIs and resources on external systems, such as Salesforce, ServiceNow, and Twitter.
+Anypoint connectors are Mule runtime engine extensions that enable you to connect to APIs and resources on external systems, such as Salesforce, ServiceNow, and Twitter.
 
 == Prerequisites
 
-Before creating an app, you must have access to the ServiceNow target resource and
-Anypoint Platform. You also need to understand how to create a Mule app using
-Design Center or Anypoint Studio.
+Before creating an app, you must have access to the ServiceNow target resource and Anypoint Platform. You also need to understand how to create a Mule app using Design Center or Anypoint Studio.
 
 == Audience
 
 * Starting user
 +
-To create your Mule app,
-see xref:servicenow/servicenow-connector-design-center.adoc[Design Center Configuration]
-or xref:servicenow/servicenow-connector-studio.adoc[Studio Configuration].
-//The xref:servicenow/servicenow-connector-config-topics.adoc[Additional Configuration Information]
-//topic helps extend your understanding of the ServiceNow connector.
-
-
+To create your Mule app, see xref:servicenow/servicenow-connector-design-center.adoc[Design Center Configuration] or xref:servicenow/servicenow-connector-studio.adoc[Studio Configuration].
 * Power user
 +
 Read the xref:servicenow/servicenow-connector-xml-maven.adoc[XML and Maven Support]
-//xref:servicenow/servicenow-connector-config-topics.adoc[Additional Configuration Information],
 and xref:servicenow/servicenow-connector-examples.adoc[Examples] topics.
 The Examples topic provides one or more use cases for using the connector.
 
 == Common Use Cases For the Connector
 
-With the ServiceNow connector, organizations can fully integrate business processes across HR, legal, procurement, operations, marketing, and facilities departments.
+ServiceNow Connector enables organizations to fully integrate business processes across HR, legal, procurement, operations, marketing, and facilities departments.
 Creating connectivity within and outside the enterprise is quick and simple with connectivity to over 120 Anypoint connectors.
 
-MuleSoft’s Anypoint ServiceNow connector allows businesses to create instant API connectivity with the ServiceNow API, allowing them to quickly and easily interface with ServiceNow from within the MuleSoft Anypoint Platform.
+Use ServiceNow Connector to create instant API connectivity with the ServiceNow API, and quickly and easily interface with ServiceNow from within Anypoint Platform.
 
 
 xref:servicenow/servicenow-connector-examples.adoc[Example] use cases includes getting a ServiceNow incident record.
 
-== Learn Using Exchange Templates and Examples
+== Using Exchange Templates and Examples
```

Comment:
Using > **_Use_** 
(Per my understanding on Kim edits, we should avoid present continuous in Titles section, like Configuring, Setting, etc)